### PR TITLE
Update view-search-usage-reports.md

### PR DESCRIPTION
**Pull request by wbaer**
Updated timeline for deprecation.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/view-search-usage-reports.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ description: "View usage reports and search reports and see how often your users
...
+>Beginning in Q4 CY2021, we'll be retiring classic tenant-wide search usage reports in SharePoint Online.  After this change, you will be able to discover site collection search usage data through the classic site collection usage reports under the **Site** > **Site Settings** > **Site collection Administration** > **Search Reports** where you can access and download usage data for the last 31 days and past 12 months. For administrators of Microsoft Search you can access the tenant usage analytics reports through the Microsoft 365 admin center under **Settings** > **Search and Intelligence** > **Insights**.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/view-search-usage-reports.md:24]:
```suggestion
>Starting in the fourth quarter of 2021, we'll be retiring classic tenant-wide search usage reports in SharePoint Online.  After this change, you will be able to discover site collection search usage data through the classic site collection usage reports under the **Site** > **Site Settings** > **Site collection Administration** > **Search Reports** where you can access and download usage data for the last 31 days and past 12 months. For administrators of Microsoft Search you can access the tenant usage analytics reports through the Microsoft 365 admin center under **Settings** > **Search and Intelligence** > **Insights**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**